README.md

```diff
@@ -68,10 +68,6 @@ console.log(output.blob()); // Blob
 > A model that outputs file data returns a `FileOutput` object by default. This is an implementation
 > of `ReadableStream` that returns the file contents. It has a `.blob()` method for accessing a
 > `Blob` representation and a `.url()` method that will return the underlying data-source.
->
-> **This data source can be either a remote URL or a data-uri with base64 encoded data. Check
-> out the documentation on [creating a prediction](https://replicate.com/docs/topics/predictions/create-a-prediction)
-> for more information.**
```

Comment:
Do we want to include a recommendation that people _don't_ use `.url()`, as it may in future return data URLs?